app/concepts/api/v1/users/registrations/operation/create.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Api::V1::Users::Registrations::Operation
+  class Create < ApplicationOperation
+    step :validate
+    step :create
+
+    private
+
+    def validate(params)
+      result = Api::V1::Users::Registrations::Contract::Validate.call(params)
```

Comment:
Ugly naming. Please use the same as concept name: `Api::V1::Users::Registrations::Contract::Create`

Reply:
Ok